data/mods/Medieval_Stuff/melee.json

```diff
@@ -0,0 +1,186 @@
+        "price": 10000,
+        "material": "aluminum",
+        "flags" : ["BASH", "SHEATH_SWORD"],
+        "techniques": "WBLOCK_1",
+        "weight": 1409,
+        "volume": 10,
+        "bashing": 11,
+        "cutting": 2,
+        "to_hit": 2,
+        "max_charges": 0,
+        "initial_charges": 0,
+        "charges_per_use": 0,
+        "turns_per_charge": 0,
+        "ammo": "NULL",
+        "revert_to": "null",
+        "category" : "weapons"
+    },
+  {
+    "id": "halberd_fake",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "halberd",
+    "description": "This is a dull, cheaply made replica of a polearm with an axe blade, a spike, and other fun things attached to a long stick.",
+    "price": 5000,
+    "material": ["wood", "aluminum"],
+    "flags": ["BASH", "REACH_ATTACK"],
+    "techniques": ["WBLOCK_1"],
+    "weight": 1644,
+    "volume": 15,
+    "bashing": 12,
+    "cutting": 2,
+    "to_hit": 2,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "category" : "weapons"
+  },
+  {
+    "id": "scimitar_fake",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "scimitar",
+    "description": "This is a dull, cheaply made replica of a curved sword, associated with various middle eastern and central asian countries.",
+    "price": 9300,
+    "material": "aluminum",
+    "techniques": ["RAPID", "WBLOCK_1"],
+    "flags" : ["BASH", "SHEATH_SWORD"],
+    "weight": 586,
+    "volume": 8,
+    "bashing": 6,
+    "cutting": 2,
+    "to_hit": 1,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "category" : "weapons"
+  },
+  {
+    "id": "ji",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "yellow",
+    "name": "ji",
+    "name_plural": "ji", "//" : "Couldn't find a good way to pluralize the transliteration.",
+    "description": "This is a bronze polearm that originated in the Shang dynasty of China, if not earlier.  It combines a spear head with the perpendicular blade of the earlier ge or dagger-axe.",
+    "price": 50000,
+    "material": ["wood", "bronze"],
+    "flags": ["CHOP", "DURABLE_MELEE", "REACH_ATTACK"],
+    "techniques": ["WBLOCK_1", "IMPALE", "DEF_DISARM"],
+    "weight": 3175,
+    "volume": 15,
+    "bashing": 8,
+    "cutting": 20,
+    "to_hit": 2
+  },
+    {
+        "type":"GENERIC",
+        "id" : "spear_dory",
+        "name" : "dory",
+        "name_plural": "dories",
+        "description" : "A well-made spear with a bronze head, Greek in origin.",
+        "weight" : 1598,
+        "to_hit" : 1,
+        "color" : "yellow",
+        "symbol" : "/",
+        "material" : ["bronze", "wood"],
+        "techniques" : ["WBLOCK_1", "RAPID", "IMPALE"],
```

Comment:
Rapid on spear technically makes sense, but it can cause problems in game due to how it "shuts down" slow creatures easily.